lib/backend/pgbk/pgbk.go

```diff
@@ -159,16 +154,18 @@ func NewWithConfig(ctx context.Context, cfg Config) (*Backend, error) {
 	log := logrus.WithField(trace.Component, componentName)
 
 	if cfg.AuthMode == AzureADAuth {
-		bc, err := pgcommon.AzureBeforeConnect(cfg.AzureClientID, log)
+		bc, err := pgcommon.AzureBeforeConnect(log)
 		if err != nil {
 			return nil, trace.Wrap(err)
 		}
 		poolConfig.BeforeConnect = bc
 	}
 
-	poolConfig.AfterConnect = func(ctx context.Context, c *pgx.Conn) error {
-		_, err := c.Exec(ctx, "SET default_transaction_isolation TO serializable", pgx.QueryExecModeExec)
-		return trace.Wrap(err)
+	if poolConfig.ConnConfig.RuntimeParams["default_transaction_isolation"] != "" {
```

Comment:
nit: `default_transaction_isolation` is repeated here 4x. Can you create a constant or use one if it's available somewhere?

Reply:
polished in 238e29e8f8